src/navigation/navigationhandler_lua.inl

```diff
@@ -25,30 +25,39 @@
 namespace {
 
 /**
- * Load a navigation state from file. The file should be a lua file returning the
- * navigation state as a table formatted as a Navigation State, such as the output files
- * of saveNavigationState. If usetimeStamp is set to true and the provided navigation
- * state has a timestamp, time will be set as well.
+ * Set the camera position by loading a navigation state from file. The file should be in
+ * json format, such as the output files of `saveNavigationState`.
+ *
+ * \param filePath the path to the file, including the file name (and extension, if it is
+ *                 anything other than `.navstate`)
+ * \param useTimeStamp if true, and the provided navigation state includes a timestamp,
+ *                     the time will be set as well.
  */
-[[codegen::luawrap]] void loadNavigationState(std::string cameraStateFilePath,
+[[codegen::luawrap]] void loadNavigationState(std::string filePath,
                                               bool useTimeStamp = false)
 {
-    if (cameraStateFilePath.empty()) {
+    if (filePath.empty()) {
         throw ghoul::lua::LuaError("Filepath string is empty");
     }
 
     openspace::global::navigationHandler->loadNavigationState(
-        cameraStateFilePath,
+        filePath,
         useTimeStamp
     );
 }
 
 /**
- * Return the current navigation state as a Lua table. The optional argument is the scene
- * graph node to use as reference frame. By default, the reference frame will picked based
- * on whether the orbital navigator is currently following the anchor node rotation. If it
- * is, the anchor will be chosen as reference frame. If not, the reference frame will be
- * set to the scene graph root.
+ * Return the current navigation state as a Lua table.
+ *
+ * By default, the reference frame will be picked based on whether the orbital navigator is
+ * currently following the anchor node rotation. If it is, the anchor will be chosen as
+ * reference frame. If not, the reference frame will be set to the scene graph root.
+ *
+ * \param frame the identifier of an optional scene graph node to use as reference frame
+ *              for the navigation state
+ *
+ * \return a Lua table representing the current [navigation state](#core_navigation_state)
```

Comment:
should we use `NavigationState` here or stick with `navigation state`?  We're currently inconsistent